Firestore/Source/API/FSTUserDataConverter.mm

```diff
@@ -427,7 +457,7 @@ - (nullable FSTFieldValue *)parseData:(id)input context:(FSTParseContext *)conte
     // If context.path is nil we are already inside an array and we don't support field mask paths
     // more granular than the top-level array.
     if (context.path) {
-      [context.fieldMask addObject:context.path];
+      context.fieldMask->push_back(*(context.path));
```

Comment:
Extra parens around `context.path`. I'm only calling you on it because you did it the other way 20 lines below :-).